torch/csrc/jit/fusers/cpu/fusion_compiler.cpp

```diff
@@ -14,19 +14,95 @@
 #include <sstream>
 #include <tuple>
 
+#if ((__GLIBC__ == 2) && (__GLIBC_MINOR__ >= 2)) || (__GLIBC__ > 2)
+#include <spawn.h>
+#ifdef POSIX_SPAWN_USEVFORK
+#define USE_POSIX_SPAWN
+#include <sys/wait.h>
+extern char **environ;
+#endif
+#endif
+
+#ifndef USE_POSIX_SPAWN
+#ifndef __APPLE__
+#include <malloc.h>
+#define USE_MALLOC_TRIM
```

Comment:
Is there any better way to check for availability of `malloc.h`? It's a GNU extension, so it's not guaranteed to be there if you e.g. use clang to build on Linux. The one above is nicer because is has a more robust check. I'd prefer to remove this otherwise.

Reply:
I'm not sure if there's a better way than guarding for glibc, so I think I'll just remove this.